os/arch/arm/src/stm32l4/stm32l4_irq.c

```diff
@@ -373,7 +373,7 @@ void up_irqinitialize(void)
    * Fault handler.
    */
 
-#ifdef CONFIG_ARM_MPU
+#ifndef CONFIG_ARMV7M_MPU
```

Comment:
is it with #ifdef?

Reply:
can you detail the problem for this PR

Reply:
I don't know why previous is wrong. Could you let know detail as @sangam-swami said?

Reply:
Oh yes!, corrected that.
It is currently using ARM_MPU config, whereas when compared to other chipset codes, cortex specific flag should be used.

Reply:
We can use both ARM_MPU and ARMV7M_MPU. No difference between them. I don't know this is necessary. If you want to be the same as others, please write that even we can use ARM_MPU, we will change it for unification in commit description.

Reply:
Yes that is the purpose, I have modified the description.